modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/VerticalCard.js

```diff
@@ -0,0 +1,129 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {ClayCardWithInfo} from '@clayui/card';
+import ClayIcon from '@clayui/icon';
+import ClaySticker from '@clayui/sticker';
+import React, {useState} from 'react';
+
+import getDataAttributes from './get_data_attributes';
+
+export default function VerticalCard({
+	actions = [],
+	componentId: _componentId,
+	cssClass,
+	description,
+	disabled,
+	displayType,
+	flushHorizontal,
+	flushVertical,
+	href,
+	imageAlt,
+	imageSrc,
+	inputName,
+	inputValue,
+	labels = [],
+	labelStylesMap: _labelStylesMap,
+	locale: _locale,
+	portletId: _portletId,
+	portletNamespace: _portletNamespace,
+	selectable,
+	selected: initialSelected,
+	stickerCssClass,
+	stickerIcon,
+	stickerImageAlt,
+	stickerImageSrc,
+	stickerLabel,
+	stickerShape,
+	stickerStyle,
+	symbol,
+	title,
+	...otherProps
+}) {
+	const [selected, setSelected] = useState(initialSelected);
+
+	const checkboxProps =
+		inputName && inputValue ? {name: inputName, value: inputValue} : null;
```

Comment:
Looking at [value attribute of checkbox](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox), it is valid to use `inputName` without `inputValue`. Value would just be `'on'` by default. On the flip side, it wouldn't make sense to have `inputValue` without `inputName`, as `name` is the identifier in form submission. Having that in mind, this should be:
```
inputName ? {name: inputName, value: inputValue} : null;
```
Probably even better would be to also allow `inputValue`, for the edge case of setting `name` dynamically before submission. Then, it would be:
```
checkboxProps={{
	name: inputName,
	value: inputValue,
}}
```
Sometimes we would send a needless empty object prop, but it would be a bit safer code.

Reply:
you're right about the value attribute for checkbox,  in this case, *React* is complaining because it gets `null` for `checkboxProps.value`

```js
checkboxProps={{
  name: null,
   value: null
}}
```

So i was thinking using 

```js
checkboxProps={{
  name: inputName ?? '',
   value: inputValue ?? '',
}}
```